packages/suite-desktop/src-electron/mocks/electron-updater.ts

```diff
@@ -0,0 +1,102 @@
+import { app } from 'electron';
+
+class CancellationToken {
+    cancelled: boolean;
+
+    constructor() {
+        this.cancelled = false;
+    }
+
+    cancel() {
+        this.cancelled = true;
+    }
+}
+
+type Events =
+    | 'checking-for-update'
+    | 'update-available'
+    | 'update-not-available'
+    | 'download-progress'
+    | 'update-downloaded'
+    | 'error';
+
+const callbacks = {
+    'checking-for-update': () => null,
+    'update-available': (_args: any) => null,
+    'update-not-available': (_args: any) => null,
+    'download-progress': (_args: any) => null,
+    'update-downloaded': (_args: any) => null,
+    error: (_args: any) => null,
+};
+
+/**
+ * The parameter `--mock-update-available=DELAY` will pretend that a new update is
+ * available for a given delay. This is disabled by default and can be enabled with
+ * any value starting from 0 (available at start) to the delay in seconds before the
+ * update should be available.
+ */
+const updateAvailableDelay = Number(
```

Comment:
nitpick: Maybe we could name this variable (and the command line switch) a bit better. It's a bit hard for me do decipher the meaning of `updateAvailableDelay`.

What about `triggerUpdateAfter` (`--trigger-update-after`)?

Feel free to RBY (resolve by yourself).

Reply:
Addressed in 4200d30.

Reply:
Dankeschön!